components/channel_header_dropdown/channel_header_dropdown_items.tsx

```diff
@@ -309,7 +310,6 @@ export default class ChannelHeaderDropdown extends React.PureComponent {
                         currentUser={user}
                     />
                     <MenuItemCloseChannel
-                        id='channelCloseChannel'
```

Comment:
same as for other ID related suggestion

Reply:
As I mentioned above, this is out of the scope of this PR.